client/state/application-passwords/actions.js

```diff
@@ -1,85 +0,0 @@
-import {
-	APPLICATION_PASSWORD_CREATE,
```

Comment:
We can now delete [all those action types](https://github.com/Automattic/wp-calypso/blob/71cbd0d9760782db728e1ef933009031ce3b10d8/client/state/action-types.ts#L51-L57).